blockchain/chain.go

```diff
@@ -862,8 +892,7 @@ func (b *BlockChain) reorganizeChain(detachNodes, attachNodes *list.List) error
 	// entails loading the blocks and their associated spent txos from the
 	// database and using that information to unspend all of the spent txos
 	// and remove the utxos created by the blocks.
-	view := NewUtxoViewpoint(b.MaxOutputsPerBlock())
-	view.SetBestHash(&oldBest.hash)
+	view := NewUtxoViewpoint()
 	for e := detachNodes.Front(); e != nil; e = e.Next() {
 		n := e.Value.(*blockNode)
```

Comment:
Elsewhere we decided to put error checking on casts. Not sure if we want to set that as a style guide. Just pointing it out.